CMakeLists.txt

```diff
@@ -61,7 +63,7 @@ endif()
 find_package(
   Boost ${BOOST_VERSION} REQUIRED COMPONENTS filesystem iostreams json
                                              serialization unit_test_framework)
-find_package(Qt6 ${QT_VERSION} REQUIRED COMPONENTS Widgets Svg)
+find_package(Qt6 ${QT_VERSION} REQUIRED COMPONENTS Widgets SvgWidgets)
```

Comment:
We use QSvgGenerator and QSvgRenderer, both of which are in Svg, not SvgWidgets.  (See https://doc.qt.io/qt-6.5/qtsvg-module.html)